sonic_platform_base/chassis_base.py

```diff
@@ -218,32 +221,38 @@ def get_component(self, index):
     def get_num_modules(self):
         """
         Retrieves the number of modules available on this chassis
+        On a SmarSwitch chassis this will be the number of DPUs.
 
         Returns:
-            An integer, the number of modules available on this chassis
+            An integer, the number of modules available on this chassis.
+            On a SmartSwitch this will be the number of DPUs
         """
         return len(self._module_list)
 
     def get_all_modules(self):
         """
-        Retrieves all modules available on this chassis
+        Retrieves all modules available on this chassis. On a SmartSwitch
+        chassis this will return all the DPUs.
 
         Returns:
             A list of objects derived from ModuleBase representing all
-            modules available on this chassis
+            modules available on this chassis. On a SmartSwitch this 
+            will be a list of DPU objects.
         """
         return self._module_list
 
     def get_module(self, index):
         """
         Retrieves module represented by (0-based) index <index>
+        On a SmartSwitch index:0 is not used, index:1 will fetch
```

Comment:
Are we starting from index 0 or 1? As the statement for get_dpu_id() is contradicting?

Reply:
Good catch! Fixed it.